tests/providers/amazon/aws/hooks/test_eks.py

```diff
@@ -1198,6 +1198,27 @@ def test_create_fargate_selectors(self, cluster_builder, selectors, expected_mes
 
 
 class TestEksHook:
+    python_executable = f"python{sys.version_info[0]}.{sys.version_info[1]}"
+    COMMAND = """
```

Comment:
Can we move the `COMMAND` declaration to the module level of the Hook and then use `EksHook.COMMAND` here, otherwise that script and the copy here may drift from each other if someone changes the script in src and forgets to update this one.